samples/bluetooth/central_ht/src/main.c

```diff
@@ -174,13 +174,13 @@ static bool eir_found(struct bt_data *data, void *user_data)
 		}
 
 		for (i = 0; i < data->data_len; i += sizeof(uint16_t)) {
-			struct bt_uuid *uuid;
+			struct bt_uuid *hts_uuid;
 			uint16_t u16;
 			int err;
 
 			memcpy(&u16, &data->data[i], sizeof(u16));
-			uuid = BT_UUID_DECLARE_16(sys_le16_to_cpu(u16));
-			if (bt_uuid_cmp(uuid, BT_UUID_HTS)) {
+			hts_uuid = BT_UUID_DECLARE_16(sys_le16_to_cpu(u16));
+			if (bt_uuid_cmp(hts_uuid, BT_UUID_HTS)) {
```

Comment:
I'm not sure I completely agree with this change. What you get from the data is "some" UUID, which could be the HRS UUID or it could be something else (that's why we perform the comparison), so calling it just `uuid` makes more sense than the more specific `hrs_uuid`. What about changing the other (presumably global?) variable name that this shadows?

Reply:
Alternatively, I guess you could also call this e.g. `found_uuid`, since it's an UUID that was found during the scan procedure.

Reply:
This also applies for the change to samples/bluetooth/central_hr/src/main.c